ci/kind/test-e2e-kind.sh

```diff
@@ -172,7 +178,11 @@ function run_test {
   rm -rf $TMP_DIR
   sleep 1
 
-  go test -v -timeout=30m antrea.io/theia/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --skip=$skiplist
+  if $coverage; then
+    go test -v -timeout=30m antrea.io/theia/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR -cover -coverpkg=antrea.io/theia/plugins/...,antrea.io/theia/pkg/... -coverprofile=.coverage/coverage-e2e.txt -covermode=atomic --skip=$skiplist
```

Comment:
$RUN_OPT missing?
$coverage_args missing?

Reply:
I don't think we have additional run options or another set of arguments to run coverage with, so might not be needed.